WD-Firmware/rtos/rtosal/config/hifive-1/FreeRTOSConfig.h

```diff
@@ -0,0 +1,168 @@
+/*
+    FreeRTOS V9.0.0 - Copyright (C) 2016 Real Time Engineers Ltd.

```

Comment:
clean and merge with rtosal.h ... align with c file name. maybe rtosal_util

Reply:
Done.
Eventually it turned out that the variants are very minor. The definition values anyway remained unchanged